### ISSUE TITLE
"<natural> and <rational>  <unit> <ingredient>"  sentence, parse misses the <rational> and <unit>

### ISSUE THREAD

**Issue by About2Golf** [strangetom/ingredient-parser#4]
I've tried various ingredient sentences and see consistent performance, so bringing this up to see if it's something that could be improved.

example 1 result:
{
        "amount": [
          {
            "quantity": "1",
            "unit": "",
            "confidence": 0.980939
          }
        ],
        "name": "and 0.5 tsp fine grain sea salt",
        "comment": null,
        "other": null,
        "sentence": "1 and 1/2 tsp fine grain sea salt"
      }

example 2 result:
{
        "amount": [
          {
            "quantity": "1",
            "unit": "",
            "confidence": 0.971023
          }
        ],
        "name": "Cassava Flour",
        "comment": "and 0.5 cups",
        "other": null,
        "sentence": "1 and 1/2 cups Cassava Flour"
      }

example 3 result:
{
        "amount": [
          {
            "quantity": "1",
            "unit": "",
            "confidence": 0.973108
          }
        ],
        "name": "coconut",
        "comment": "and 0.333 cups unsweetened finely shredded",
        "other": null,
        "sentence": "1 and 1/3 cups unsweetened finely shredded coconut"
      }

example 4 result:
{
        "amount": [
          {
            "quantity": "1",
            "unit": "",
            "confidence": 0.968139
          }
        ],
        "name": "dark chocolate morsels",
        "comment": "and 0.5 cups",
        "other": null,
        "sentence": "1 and 1/2 cups dark chocolate morsels"
      }

**Comment by strangetom** [comment]
Thanks for the report, good catch. I don't think this will be too difficult to fix, I'll take a look in the next few days.

**Comment by About2Golf** [comment]
Sounds good, thanks for taking a look!

**Comment by strangetom** [comment]
There's a fix for this in the prep_labels branch (7759ba2e56f05393b4b0300c3799b889cc90fff8), so it will be in the next release.

As an example, the sentence `1 and 1/2 cups Cassava Flour`, will give a result that looks like:
```python
{
  'name': 'Cassava Flour',
   'amount': [
    {
      'quantity'='1.5', 
      'unit'='cups',
    }
   ],
   'preparation': None,
   'comment': None,
   'other': None,
   'sentence': '1 and 1/2 cups Cassava Flour'
}
```

**Comment by About2Golf** [comment]
Excellent, thank you!
